src/Objs/Homepage/HomepageEditingConfig.js

```diff
@@ -62,6 +62,11 @@ Scrivito.provideEditingConfig("Homepage", {
         "Removes the header navigation and only centers the logo instead. Default: No",
       values: [{ value: "yes", title: "Yes" }, { value: "no", title: "No" }],
     },
+    cookieConsentLink: {
+      title: "Cookie privacy policy link",
```

Comment:
I suggest to change the title to `Cookie consent link`.

See https://github.com/Scrivito/scrivito_example_app_js/pull/100#discussion_r213265683 for details.